src/components/Header/NavComponents/userprofile.tsx

```diff
@@ -4,10 +4,21 @@ import {
   Center,
   Badge,
   useColorModeValue as mode,
+  Button,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverBody,
+  Stack,
+  PopoverArrow,
 } from "@chakra-ui/react";
 import { bigNumberToString } from "../../../utils/fixedPoint";
 import { useAmountAvailableToStake } from "../../../queries/amountAvailableToStake";
 import { useAppWeb3 } from "../../../hooks/appWeb3";
+import { useChainAddresses } from "../../../utils/chainAddresses";
+import { internalAddressesPerNetworkId, } from "../../../utils/contracts/contractAddresses/internalAddresses";
+
+declare let window: any;
```

Comment:
Don't use `window`. You won't need it, if you're using the web3 connectors as an abstraction.

Reply:
I removed the use of window